src/Libraries/CoreNodes/DSCoreNodes.Migrations.xml

```diff
@@ -38,4 +38,20 @@
       </attribute>
     </additionalAttributes>
   </priorNameHint>
+  <priorNameHint>
+    <oldName>DSCore.Formula.Evaluate</oldName>
+    <newName>DSCore.Math.EvaluateFormula</newName>
+  </priorNameHint>
+  <priorNameHint>
+    <oldName>DSCore.Logic.Xor</oldName>
+    <newName>DSCore.Math.Xor</newName>
+  </priorNameHint>
+  <priorNameHint>
+    <oldName>DSCore.Logic.And</oldName>
+    <newName>DSCore.Math.And</newName>
+  </priorNameHint>
+  <priorNameHint>
+    <oldName>DSCore.Logic.Or</oldName>
+    <newName>DSCore.Math.Or</newName>
```

Comment:
They do exist in `CoreNodeModels.Logic.`
https://github.com/DynamoDS/Dynamo/blob/master/test/core/customast/and-test.dyn#L4